src/main/java/hudson/plugins/ws_cleanup/RemoteCleaner.java

```diff
@@ -47,6 +48,17 @@
             TaskListener listener,
             Run<?, ?> build
     ) {
+        try {
+            EnvVars vars = build.getEnvironment(listener);
+
+            for (int i = 0; i < patterns.size(); i++) {
+                patterns.set(i, new Pattern(vars.expand(patterns.get(i).getPattern()), patterns.get(i).getType()));
+            }
+            externalDelete = vars.expand(externalDelete);
+        } catch (Exception e) {
```

Comment:
That was to satisfy an error that was thrown when using `EnvVars vars = build.getEnvironment(listener);` that complained there wasn't an exception catch.